crdppf/templates/derived/init.js

```diff
@@ -29,9 +29,9 @@ Crdppf.layers = [
             nbcols = len(layer)
         %>
             {
-            % for key in layer:
+            % for key in layer :
```

Comment:
I don't really get why you added a space everywhere before `:` in this file...

Python doesn't add a space before `:` only after...